components/Subtitle.js

```diff
@@ -0,0 +1,7 @@
+export default function Subtitle({ children }) {
+  return (
+    <h4 className="subtitle">
+      {children}
+    </h4>
+  );
+}
```

Comment:
What is the criteria to define this subtitle component as a h4? Does it make sense based on the hierarchy?

Reply:
This is based on the style guide page in Figma, there are styles for each typography